CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+
+- Support row filters and column masks in Rego rules ([#559]).
```

Comment:
I think we should mark this as breaking, as users of 442 need to adopt their rego rules when updating, WDYT?

Reply:
@siegfriedweber

Reply:
This is actually not breaking. If an OPA rule is queried which is not defined then it is "undefined". This is the same behavior if the rule is defined but the condition is not met.

The response from OPA is in both cases:

```HTTP
HTTP/1.1 200 OK
Content-Type: application/json
Vary: Accept-Encoding
Date: Wed, 27 Mar 2024 10:13:44 GMT
Content-Length: 55

{"decision_id":"facecbef-f851-40e7-afce-c46cf45b9f2c"}
```

That is, if the entry points for row filters and column mask are not defined then the behavior is the same if these entry points are defined but no row filters and column masks are defined in the policies.

Reply:
Even better, awesome! Thanks for looking it up